### PR TITLE
Docs enhancements

### PR DESCRIPTION
**Pull request by maringerov**
Hi guys, first of all - great project! I started using Radium recently and I am quite happy with the results. While getting to know how to use it I stumbled upon a few minor things I thought could be improved to the docs. Here are some suggestions I have that may make it even easier to get started with Radium :)

### CODE REVIEW

File: docs/guides/README.md [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
 
+First, require or import Radium at the top of component file:
+
+```as
+var Radium = require('radium');
+
+// or
+import Radium from 'radium'
+
+//If you want to use the <Style /> component you can do
```
Comment (ianobermiller) [inline docs/guides/README.md]: space before If

File: docs/guides/README.md [hunk 2]
```diff
@@ -283,3 +313,7 @@ Is equivalent to the following CSS (note that the order is reversed):
   background: rgba(255, 255, 255, .5);
 }
 ```
+
+## Style selector
```
Comment (ianobermiller) [inline docs/guides/README.md]: Small updates.

```md
## `<Style>` component

Want to add css selectors within your component? Need to add styles to the `html` and `body` elements or group selectors (e.g. ```h1, h2, h3```) that share properties? Radium has you covered with the `<Style />` component. Read how to use it in the [API docs](https://github.com/FormidableLabs/radium/tree/master/docs/api#style-component).
```

**Comment by ianobermiller** [comment]
Great, thanks for your changes! Just a couple little updates and it is ready to merge.

**Comment by maringerov** [comment]
Hi @ianobermiller , just pushed the changes ;) Thanks!

**Pull request merged**